bootstrap/interfaces/mocks/SecretProvider.go

```diff
@@ -96,3 +96,27 @@ func (_m *SecretProvider) StoreSecret(path string, secrets map[string]string) er
 
 	return r0
 }
+
+// HasSecret provides a mock function with given fields: path, keys
```

Comment:
Did you install and use the latest `mockery`?

 I expected the first line to change show latest version and the newer version adds some other code.